contrib/grimshot

```diff
@@ -19,29 +19,63 @@
 ## `grimshot` - usage
 ## `grimshot check` - verify if tools are installed
 
+getTargetDirectory() {
+  test -f ${XDG_CONFIG_HOME:-~/.config}/user-dirs.dirs && \
+    source ${XDG_CONFIG_HOME:-~/.config}/user-dirs.dirs
+
+  echo ${XDG_SCREENSHOTS_DIR:-${XDG_PICTURES_DIR:-$HOME}}
+}
+
+if [ $1 == "--notify" ]; then
+  NOTIFY=yes
+  shift 1
+else
+  NOTIFY=no
+fi
+
 ACTION=${1:-usage}
 SUBJECT=${2:-screen}
-FILE=${3:-$(xdg-user-dir PICTURES)/$(date +'Grimshot %Y-%m-%d %H-%M-%S.png')}
-if [ "$ACTION" = "usage" ] ; then
+FILE=${3:-$(getTargetDirectory)/$(date -Ins).png}
+
+if [ "$ACTION" != "save" ] && [ "$ACTION" != "copy" ] && [ "$ACTION" != "check" ]; then
   echo "Usage:"
-  echo "  grimshot copy|save win|screen|output|area [FILE]"
-  echo "Troubleshoot:"
+  echo "  grimshot [--notify] (copy|save) [win|screen|output|area|window] [FILE]"
   echo "  grimshot check"
+  echo "  grimshot usage"
+  echo ""
+  echo "Commands:"
+  echo "  copy: Copy the screenshot data into the clipboard."
+  echo "  save: Save the screenshot into a regular file."
```

Comment:
i'd go with copy and save _to_ vs _into_

Reply:
Updated "Save [..] to a regular file.  
`into the clipboard` makes more sense I think.